packages/example-getting-started/docs/4-todo-model.md

```diff
@@ -29,17 +39,20 @@ import {Bar} from './models/bar.model';
 import {Baz} from './models/baz.model';
 ```
 
-In our Todo model, we'll create a basic representation of what would go in
-a Todo list. Our model will include:
+For our Todo model to represent our Todo instances, it will need:
```

Comment:
I think before we get into what our `Todo` model would look like, we should describe at a top-level what the function of `Todo` is;  Something like this: 
```markdown
We want an entry of Todo to contain:
- a name of the task
- a description of the task
- a checkmark for its completion status
```

Reply:
ping